src/app/api/common/delegates/getDelegates.ts

```diff
@@ -49,10 +49,10 @@ export async function getDelegatesForNamespace({
         case "weighted_random":
           return prisma.$queryRawUnsafe<DelegatesGetPaylod[]>(
             `
-            SELECT *, setseed($1)::Text
+            SELECT *
             FROM ${namespace + ".delegates"}
             WHERE voting_power > 0
-            ORDER BY -log(random()) / voting_power
```

Comment:
let's keep the weighted random functionality here please

Reply:
@stepandel , I think I see what @ferrodri  is doing here. The ordering is based on md5 value, which is effectively random but consistent with the seed. I did some basic QA and the values look good.